communitiesuk/funding-service-design-application-store
# FS-2617: Use LRU cache on round name

### Change description

Cache round_name request as we don't need call it every time, since it'll always be the same
Add TODO comment to clean up round_name function after we check it's not being used elsewhere


### How to test

Load the front-end on an account with more than 1 application, view the logs on application-store, observe that we're now only calling the fund-store once, instead of once PER application.
